gcloud/datastore/client.py

```diff
@@ -441,8 +441,33 @@ def transaction(self):
 
     def query(self, **kwargs):
         """Proxy to :class:`gcloud.datastore.query.Query`.
-
         Passes our ``project``.
+
+        Using query to search a datastore::
```

Comment:
This was in `connection.py`, I think it belongs here since `def query` is here? I can remove it form `connection.py` though if you think that would be better?

Also, `Connection` isn't linked in the top nav for the docs so I wanted to bring this up a level.